### PR TITLE
keyword RVWVD to specify vaporized water-gas ratio  vs depth tables

### PR DESCRIPTION
**Pull request by plgbrts**

### CODE REVIEW

File: opm/input/eclipse/EclipseState/Tables/RvwvdTable.hpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*
+  Copyright (C) 2014 by Andreas Lauser
+
```
Comment (goncalvesmachadoc) [inline opm/input/eclipse/EclipseState/Tables/RvwvdTable.hpp:3]: Please, check that all copyrights are updated.

**Comment by plgbrts** [comment]
This PR goes along with https://github.com/OPM/opm-simulators/pull/4034.
An additional item is added to the `EQUIL` keyword. With the last item (12-th)  the initialization option can be set for vaporized water in gas. If  the value is non-positive initialization is based on saturated water-gas ratio and if positive initialization use the new `RVWVD` table.
An example `EQUIL` record looks like
```
EQUIL
--  1        2       3         4      5         6      7    8    9    10  11 12
-- DATUM     DATUM   OWC       PCOW   GOC       PCGO   RS   RV	 N    -   -  RVW
   4660.6875 410.0   4660.6875 0.0    4660.6875 0.0     1   1    1*   1*  1* 1  / 
```
Note: the items 10 and 11 are not implemented.

**Comment by goncalvesmachadoc** [comment]
Food for thought. Shouldn't we reserve 12 for RSW and make RVW number 13, to follow the same order as 7 RS, 8 RV?

**Review by bska — APPROVED** [review]
This looks good to me and I'd be happy to merge this.  That said, I'd really like for there to be a set of unit tests for the Rvww table code to guard against problems during future maintenance.  Maybe we can add that in a follow-up PR?

**Comment by bska** [comment]
jenkins build this opm-simulators=4034 please

**Comment by goncalvesmachadoc** [comment]
> This looks good to me and I'd be happy to merge this. That said, I'd really like for there to be a set of unit tests for the Rvww table code to guard against problems during future maintenance. Maybe we can add that in a follow-up PR?

Thank you for reviewing it. I will take care of the follow-up PR.

**Comment by bska** [comment]
> > This looks good to me and I'd be happy to merge this. That said, I'd really like for there to be a set of unit tests for the Rvww table code to guard against problems during future maintenance. Maybe we can add that in a follow-up PR?
> 
> Thank you for reviewing it. I will take care of the follow-up PR.

Very good.  I'll merge this and its downstream companion PR into the master branches now.

**Pull request merged**